bundle/build

```diff
@@ -7,6 +7,11 @@ source "$(dirname "${BASH_SOURCE[0]}")"/vars
 
 cd "$(dirname "$0")"
 
+CONMON_VERSION=v2.0.11
+RUNC_VERSION=v1.0.0-rc10
+CNI_PLUGINS_VERSION=v0.8.5
+CRICTL_VERSION=v1.17.0
```

Comment:
I'm going to give this the lgtm, but can we source these from `scripts/versions` somehow?

Reply:
Ah yes I’ll propose this in a PR next week :)